packages/tas_keybind/init.lua

```diff
@@ -0,0 +1,79 @@
+tas = tas or {}
+tas._oldControls = {}
+
+---@alias tas.KeyListenerCallback fun(player: unknown, pressed: boolean):void
+---@alias Set<T> table<T, boolean>
+
+---@type table<tas.CONTROLS, Set<tas.KeyListenerCallback>>
+tas.keyListeners = {}
```

Comment:
I feel like both of these tables shouldn't be exposed to the global scope. Is there a reason they're not locals?

Reply:
No idea